dora/core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -6821,6 +6270,7 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.CLIENT)
           .build();
+  // used by {@link alluxio.conf.PropertyKey.Template.USER_NETWORK_FLOWCONTROL_WINDOW}
```

Comment:
thanks for adding these comments. Ideally these are added in a separate PR but I'm fine adding them here to save your labor.